lib/simp/beaker_helpers.rb

```diff
@@ -828,7 +828,8 @@ def rhel_rhsm_subscribe(suts, *opts)
 
       if os == 'RedHat'
         unless rhsm_opts[:username] && rhsm_opts[:password]
-          fail("You must set BEAKER_RHSM_USER and BEAKER_RHSM_PASS environment variables to register RHEL systems")
+          warn("BEAKER_RHSM_USER and/or BEAKER_RHSM_PASS not set on RHEL system.", "Assuming that subscription-manager is not needed. This may prevent packages from installing")
```

Comment:
This is fine. Users will probably miss the message in the noise but, in the end, it'll work or it won't and there will at least be a warning to see.